src/ipc/capnp/protocol.cpp

```diff
@@ -96,6 +90,7 @@ class CapnpProtocol : public Protocol
     Context m_context;
     std::thread m_loop_thread;
     std::optional<mp::EventLoop> m_loop;
+    std::optional<mp::EventLoopRef> m_loop_ref;
```

Comment:
In f8fd3959d51f6f80b428828c4fd965e06a8fa19e _ipc: Use EventLoopRef instead of addClient/removeClient_: if you have to retouch it would be good to document `m_loop` and `m_loop_ref`.